src/website/app/demos/Card/index.js

```diff
@@ -28,35 +28,57 @@ const cardBlockDoc = require('!!react-docgen-loader!../../../../Card/CardBlock')
 const cardImageDoc = require('!!react-docgen-loader!../../../../Card/CardImage');
 const cardTitleDoc = require('!!react-docgen-loader!../../../../Card/CardTitle');
 
+import bestPractices from './bestPractices';
+
 export default [
   {
+    bestPractices: bestPractices.card,
     componentTheme: cardComponentTheme,
     doc: cardDoc,
     examples: cardExamples,
     slug: 'card',
-    title: 'Card'
+    title: 'Card',
+    whenHowToUse: `Cards are used when you need to group different kinds of elements with actions attached.
+Cards are used to contain content of variable length that might not fit neatly into a table row or grid cell.
+They're often used to show a collection of related but different types of information.
+
+Strive to limit the actions linked from the card to one main call to action. Don't put too much information in a single card.
```

Comment:
I don't think actions (aside from when the entire Card is clickable) should be included in this component or its docs yet. But even so, "the actions linked from the card" is a confusing phrase that conflates actionable Buttons with Links.

Reply:
addressed